pkg/pillar/cmd/agent.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"flag"
+	"github.com/lf-edge/eve/pkg/pillar/agentlog"
+	"github.com/lf-edge/eve/pkg/pillar/pidfile"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// AgentBase - Interface for all agents to use the AgentBase.
+// Each agent is expected to have a Context that implements
+// this interface.
+type AgentBaseIntf interface {
```

Comment:
The naming here is not in line with normal go practices. Normally you name something just `<thing>`, not `<thing>Intf` (e.g. `AgentBase` and not `AgentBaseIntf`). You even see it instinctively, as you put the comment as `AgentBase`. As it is, lint will complain about this.